app/views/custody_suite/defence_requests/form_partials/_detainee_details.html.erb

```diff
@@ -14,6 +14,7 @@
 <% end %>
 
 <%= f.text_field_input(:detainee_address,
+    multiline: 5,
```

Comment:
I actually did find this more accurate, because from the abstraction perspective, we don't mind which html element is going to be rendered, just that it has multiple lines. 
But I'll change this.